snmp/tests/test_e2e.py

```diff
@@ -34,8 +34,9 @@ def _build_device_ip(container_ip, last_digit='1'):
 @common.python_autodiscovery_only
 def test_e2e_python(dd_agent_check):
     metrics = common.SUPPORTED_METRIC_TYPES
-    instance = common.generate_container_instance_config(metrics)
-    aggregator = dd_agent_check(instance, rate=True)
+    config = common.generate_container_instance_config(metrics)
```

Comment:
(nit, very nit) should you rename this to `test_python_e2e` ?

Reply:
Good idea. Renamed here: https://github.com/DataDog/integrations-core/pull/8708/commits/7bc1d3506aa3c32e0b9d91eddb3473134f62b59a